polyfill/spatial-navigation-polyfill.js

```diff
@@ -227,7 +227,6 @@
       if (!createSpatNavEvents('beforefocus', bestCandidate, null, dir))
         return true;
 
-      bestCandidate.focus({preventScroll:true});
```

Comment:
Did you check action? How do you move the focus?

Reply:
Oh!! :fearful: I must have been in a daze when I did this fix!
Thank you for pointing this out! I fixed this with https://github.com/WICG/spatial-navigation/commit/1272da62356be0231c42efb50f855d6097b0b3fe.